en/developer_guide/navigation_panel.rst

```diff
@@ -41,14 +41,58 @@ The menu link must have the following attributes :
 * id : merge of the "nav-" string and the name of the strategy as returned by
   ``NavigationPanelInterface::getName()``, for instance "nav-user"
 
-It can also have data-* attributes that can help you to define custom properties to be used in a Backbone route,
-for example :
+Menu and data table
+~~~~~~~~~~~~~~~~~~~
+
+Many panel entries allow to consult records formated in a data table.
+The columns of this data table can be configured: name of the column, visible or not by default, type of search field...
+The data table view is based on `DataTables library`_.
+So all options coming from this library are allowed in the definition of the columns.
+
+Here is an example of yml configuration:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.navigation_panel.trashcan.parameters :
+        -
+          name : name
+          title : open_orchestra_backoffice.table.trashcan.name
+          visible : true
+          activateColvis : true
+          searchField : text
+
+The parameter ``open_orchestra_backoffice.navigation_panel.trashcan.parameters`` is injected to panel strategy whith a translator.
```

Comment:
Is this used only for the trashcan strategie which require a specific configuration, or it is the same for all strategies ? I mean is this parameter specifically injected via the trashcan service definition, or is it injected via a generic mechanism applied to each strategie ? If yes, I assume the naming of the parameter and the subbarameter are importants: what is the 'format' ?